addons/l10n_vn/data/account_tax_data.xml

```diff
@@ -149,4 +222,199 @@
+        <field name="type_tax_use">purchase</field>
+        <field name="tag_ids" eval="[(6,0,[ref('tax_tag_import_0')])]"/>
+        <field name="tax_group_id" ref="tax_group_0"/>
+    </record>
+    <!-- for sales -->
+    <record id="tax_sale_vat_exemption" model="account.tax.template">
+        <field name="chart_template_id" ref="vn_template" />
+        <field name="name">VAT Exemption</field>
+        <field name="description">Sales VAT Exemption</field>
+        <field name="amount" eval="0" />
+        <field name="amount_type">percent</field>
+        <field name="type_tax_use">sale</field>
+        <field name="tag_ids" eval="[(6,0,[ref('tax_tag_sale_exemption'),ref('tax_tag_sale_exemption')])]"/>
+        <field name="tax_group_id" ref="tax_group_exemption"/>
+    </record>
+    <record id="tax_sale_special_consumption_10" model="account.tax.template">
+        <field name="chart_template_id" ref="vn_template" />
+        <field name="name">Special consumption tax 10%</field>
+        <field name="description">Special consumption tax 10%</field>
+        <field name="amount" eval="10" />
+        <field name="amount_type">percent</field>
+        <field name="account_id" ref="chart3332" />
+        <field name="refund_account_id" ref="chart3332" />
+        <field name="type_tax_use">sale</field>
+        <field name="tag_ids" eval="[(6,0,[ref('tax_tag_sale_special_consumption')])]"/>
+        <field name="tax_group_id" ref="tax_group_10"/>
+    </record>
+    <!-- for foreign export -->
+    <record id="tax_foreign_import_10" model="account.tax.template">
+        <field name="chart_template_id" ref="vn_template" />
+        <field name="name">Import tax 10%</field>
+        <field name="description">Import tax 10%</field>
+        <field name="amount" eval="10" />
+        <field name="amount_type">percent</field>
+        <field name="account_id" ref="chart3333" />
+        <field name="refund_account_id" ref="chart3333" />
+        <field name="type_tax_use">purchase</field>
+        <field name="tag_ids" eval="[(6,0,[ref('tax_tag_foreign_import_10')])]" />
+        <field name="tax_group_id" ref="tax_group_10"/>
+    </record>
+    <record id="tax_foreign_import_5" model="account.tax.template">
+        <field name="chart_template_id" ref="vn_template" />
+        <field name="name">Import tax 5%</field>
+        <field name="description">Import tax 5%</field>
+        <field name="amount" eval="5" />
+        <field name="amount_type">percent</field>
+        <field name="account_id" ref="chart3333" />
+        <field name="refund_account_id" ref="chart3333" />
+        <field name="type_tax_use">purchase</field>
+        <field name="tag_ids" eval="[(6,0,[ref('tax_tag_foreign_import_5')])]" />
+        <field name="tax_group_id" ref="tax_group_5"/>
+    </record>
+    <record id="tax_foreign_import_0" model="account.tax.template">
+        <field name="chart_template_id" ref="vn_template" />
```

Comment:
Should we define Import Tax?

Probably it is easier to handle import tax as a product and have a separate Supplier invoice to the Custom Tax Authorities.

Also, to do this, the account 3333 should be considered as Payable (not Current Liabilities)


Similar cases are:

 - Payable for Social Insurance
 - Payable for all taxes

Actually, the account type should be Current Liabilities, but what is the best way to handle this in Odoo? Because in Vendor Bill and Supplier Receipt, we can only accept an account of type Payable, but not Current Liabilities.